bin/nyc.js

```diff
@@ -117,7 +117,7 @@ if (process.env.NYC_CWD) {
   } else if (argv._.length) {
     // wrap subprocesses and execute argv[1]
     var nyc = (new NYC())
-    nyc.cleanup()
+    nyc.reset()
```

Comment:
> Where did it call cleanup()? 

Right there in that diff. Now instead it calls `reset`, which deletes and recreates the directory.

The issue is that `all` immediately and synchronously writes a "preample" for every included file to disk, before continuing with the test, so the folder needs to exist, even though we haven't yet forked a process.